core/libs/session/session.php

```diff
@@ -44,6 +44,18 @@ public static function set($index, $value, $namespace = 'default')
         $_SESSION[self::SESSION][APP_PATH][$namespace][$index] = $value;
     }
 
+    /**
+     * Añade un elemento a una matriz con el indice de sesion indicado
+     *
+     * @param string $index
+     * @param mixed  $value
+     * @param string $namespace
+     */
+    public static function add($index, $value, $namespace='default')
```

Comment:
__[PHPMD]__

add accesses the super-global variable $_SESSION.


<!-- Comment by [SideCI](https://sideci.com) -->